docs/dev/contributing.rst

```diff
@@ -120,3 +120,38 @@ hasn't been reported before. Duplicate bug reports are a huge drain on the time
 of other contributors, and should be avoided as much as possible.
 
 .. _GitHub issues: https://github.com/pypa/pipenv/issues
+
+Run the tests
+-------------
+
+Three ways of running the tests are as follows:
+
+1. ``make test`` (which uses ``docker``)
+2. ``./run-tests.sh``
+3. Using pipenv::
+
+    pipenv install --dev
+    pipenv run py.test
```

Comment:
I believe `pytest` is recommended over `py.test` by Pytest folks now. Also you might want to consider Windows for these instructions :) Most of these won’t work on Windows.

Reply:
I changed to `pytest` as suggested.

OTOH, I am not inclined (or able) to also document what is needed for Windows. I agree that the example config instructions won't work for most people. They are explicitly an example of what might be required, and I think they will be helpful even for people who have to modify them. I also think more examples from more platforms would be good.